consensus_messages.proto

```diff
@@ -41,3 +41,16 @@ message ConsensusMessage {
   // payload of message
   ConsensusMessagePayload payload = 2;
 }
+
+message ConsensusMessage {
+   ConsensusMessageHeader header = 1;
+   
+   // Signature of header by voter private key
+   bytes header_signature = 2;
+}
+
+message ConsensusSeal { 
+  // That should be equal to header_signature of previous block.
+  bytes previous_id = 1;
```

Comment:
There is no need to explicitly store that. It is included to block headers.

Reply:
When we will do catch-up, we will send and receive ConsensusSeal message, and if we remove this field there is no other way to check that this block correspond to given cert-votes besides parsing every cert-vote.